wallet-core/wallet-core-usage.md

```diff
@@ -164,6 +164,68 @@ For more details on Ethereum transactions, check the Ethereum documentation.  A
 * https://kauri.io/article/7e79b6932f8a41a4bcbbd194fd2fcc3a/v2/ethereum-101-part-4-accounts-transactions-and-messages
 * https://github.com/ethereumbook/ethereumbook/blob/develop/06transactions.asciidoc
 
+### Binance Chain (BNB) Transaction Signing
+
+Binance Chain is built upon [cosmos-sdk](https://github.com/cosmos/cosmos-sdk), instead of `Message`, transaction in Binance Chain is called `Order`, [Binance.proto](https://github.com/trustwallet/wallet-core/blob/master/src/proto/Binance.proto#L144) shows all the orders that Wallet Core currently supports.
+
+To sign a order, you need to use `BinanceSigningInput`: 
+
+Field | Sample value | Description
+---|---|---
+chainID | "Binance-Chain-Nile" | Network id, use "Binance-Chain-Tigris" for mainnet (see [node-info](https://dex.binance.org/api/v1/node-info) api)
```

Comment:
can you change:
"Binance-Chain-Nile" => `Binance-Chain-Nile`
"Binance-Chain-Tigris" => `Binance-Chain-Tigris`